src/redux/features/books/booksSlice.js

```diff
@@ -1,18 +1,44 @@
 import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuidv4 } from 'uuid';
 
 const initialState = {
-  books: [],
+  books: [
+    {
+      itemId: uuidv4(),
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      itemId: uuidv4(),
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      itemId: uuidv4(),
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
 };
 
 const booksSlice = createSlice({
   name: 'books',
   initialState,
   reducers: {
     addBook: (state, action) => {
-      state.books.push(action.payload);
+      const newBook = {
+        itemId: uuidv4(),
+        title: action.payload.title,
+        author: action.payload.author,
+        category: action.payload.category,
+      };
+      state.books.push(newBook);
```

Comment:
- [ ] The addBook function is not working and I suspect it is from here, I will encourage you to change this to `state.books = [...state.books, action.payload]` and by that, we are not mutating the state and we are taking advantage of the spread operator to add to the books present there before. As it stands, I cant add to the books. Once I try to, it loads the initial state data only.